rotkehlchen/accounting/transactions.py

```diff
@@ -25,15 +26,15 @@ def evm_events_iterator(
     Takes associated event as an argument to be able to log it in case of errors.
     """
     for event in events_iterator:
-        if not isinstance(event, EvmEvent):
+        if not isinstance(event, EvmEvent) and not isinstance(event, HistoryEvent):  # TODO: undo. Is a temporary hack  # noqa: E501
             log.error(
                 f'At accounting for tx_event {associated_event.notes} with hash '
                 f'{associated_event.tx_hash.hex()} we expected to take an additional '
```

Comment:
Won't this line fail for history events?

Reply:
:thinking: where and why? What will be wrong?

Reply:
Ah, you mean `{associated_event.tx_hash.hex()}`? Yeah, probably it will. Fixed it